frontend/sass/laletterbuilder/styles.scss

```diff
@@ -192,6 +192,64 @@ $jf-navbar-height: 70px;
   }
 }
 
+// CHOOSE LETTER STYLING OVERRIDES
+
+.jf-la-letter-card {
+  border: 1px solid $new-justfix-black;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  max-width: 654px;
+
+  .content {
+    padding: 24px 24px 0px 24px;
+    margin-bottom: 0px;
```

Comment:
Might be good to write these px amounts in terms of rem instead of px for consistency and accessibility. Not a blocker tho!

Reply:
done